app.py

```diff
@@ -8,6 +8,7 @@ def say_quote():
 	quotes = ['I never wanted you to come back to Gotham. I always knew there was nothing for you here, except pain and tragedy. And I wanted something more for you than that. I still do.', 'We burned the forest down.', 'Endure, Mister Wayne. Take it. They will hate you for it, but that is the point of the Batman. He can be the outcast, he can make the choice that no one else can make, the right choice.', 'Some men just want to watch the world burn.', 'I have sewn you up, I have set your bones, but I will not bury you. I have buried enough members of the Wayne family.', 'He is not being a hero. He is being something more.', 'I am so sorry. I failed you. You trusted me and I failed you', 'Rachel believed in what you stood for, what we stand for. Gotham needs you.']
 	quote_nr = random.randrange(7)
 	voice.system_say(quotes[quote_nr])
+helooooo
```

Comment:
Codacy suggests to change to: (UI created)
```suggestion
helo
```